src/HotChocolate/ElasticSearch/src/Data.ElasticSearch.Driver/Driver/ContainsOperation.cs

```diff
@@ -0,0 +1,20 @@
+namespace HotChocolate.Data.ElasticSearch;
+
+public class ContainsOperation : ILeafSearchOperation
+{
+    public ContainsOperation(
```

Comment:
Can we rename this to WildcardOperation to align with elastic?

Reply:
Sure thing! 
At the time of coding, it got me thinking for a while, whether I should take HotChocolate naming conventions as the first priority or elastic queries and I ended up choosing contains. ;)
I'll rename this and consider paying more attention to elastic naming conventions in the future.

Reply:
The idea is that the "Operations" in the core package are as close as possible to the elastic search representation. It should really just be a 1:1 mapping.